src/pybind/mgr/cephadm/module.py

```diff
@@ -2859,6 +2866,36 @@ def set_prometheus_access_info(self, user: str, password: str) -> str:
         self.set_store(PrometheusService.USER_CFG_KEY, user)
         self.set_store(PrometheusService.PASS_CFG_KEY, password)
         return 'prometheus credentials updated correctly'
+    
+    @handle_orch_error
+    def set_prometheus_targets(self, url: str) -> str:
```

Comment:
NIT: the function name is plural, but only accepts a single url. Same for the remove version.